src/connector/src/sink/print.rs

```diff
@@ -0,0 +1,201 @@
+// Copyright 2022 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::HashMap;
+
+use async_trait::async_trait;
+use itertools::join;
+use risingwave_common::array::{Op, StreamChunk};
+use risingwave_common::catalog::Schema;
+use risingwave_common::types::{DatumRef, ScalarRefImpl};
+
+use crate::sink::{Result, Sink};
+
+pub const PRINT_SINK: &str = "print";
+
+#[derive(Clone, Debug)]
+pub struct PrintConfig {
+    pub prefix: String,
+    pub suffix: String,
+}
+
+impl PrintConfig {
+    pub fn from_hashmap(values: HashMap<String, String>) -> Result<Self> {
+        let prefix = values
+            .get("prefix")
+            .expect("prefix is required")
```

Comment:
why the field is required

Reply:
I think so. I've optioned the fields in ConsoleConfig.